templates/resources/accessanalyzer_analyzer.md.tmpl

```diff
@@ -0,0 +1,34 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+{{ if .HasExample -}}
```

Comment:
I think there is a bug in `terraform-plugin-docs`, the `.HasExample` function only seems to return true if there is a `resource.md` file in the examples folder. That my be by design. So in this case where you have multiple example files it doesn't really work as expected. Will push up some changes to resolve this.